docs/building_on_the_vm.md

```diff
@@ -0,0 +1,31 @@
+# Building on the VM
```

Comment:
"Building on the VM" could mean a lot of things. What do you think about a more specific title, such as "Building images within the VM"?

Reply:
Yeah, that is probably a better title